HW_1/task_4.py

```diff
@@ -0,0 +1,18 @@
+# Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, 
+# если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+
+# *Пример:*
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+N = int(input('Введите количество долек в ширину: '))
+M = int(input('Введите количество долек в длинну: '))
+K = int(input('Сколько долек хотим отломить: '))
+
+if K < (N*M):
+
+    if K == N or K == M or K % N == 0 or K % M == 0:
```

Comment:
Какой это очень сложное условие. Без первых 2-х можно обойтись.
```suggestion
    if K % N == 0 or K % M == 0:
```